src/utils/evm/pause.ts

```diff
@@ -0,0 +1,42 @@
+import { EthereumConfig } from '@buildwithsygma/sygma-sdk-core'
+import { Wallet } from 'ethers'
+import { RpcEndpoints } from '../../types'
+import { EVM_BLOCK_CONFIRMATIONS } from '../../constants'
+import { initEvmBridgeInstance } from '.'
+
+export async function sendEvmPauseTransaction(
+  ethereumConfigs: Array<EthereumConfig>,
+  rpcEndpoints: RpcEndpoints,
+  wallet: Wallet
+): Promise<void> {
+  await Promise.all(
+    ethereumConfigs.map(async (network) => {
+      const bridgeInstance = initEvmBridgeInstance(
+        rpcEndpoints[network.chainId],
+        network,
+        wallet
+      )
+
+      try {
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-call
+        const tx = await bridgeInstance.adminPauseTransfers({
+          // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access
+          from: wallet.address,
+        })
+        console.log(
```

Comment:
Is there some equivalent to `prompt` just for informing users?

Reply:
Note sure what you mean.